fontbe/src/orchestration.rs

```diff
@@ -120,7 +127,9 @@ impl Identifier for WorkId {
             WorkId::Hhea => "BeHhea",
             WorkId::Hmtx => "BeHmtx",
             WorkId::Hvar => "BeHvar",
-            WorkId::Kerning => "BeKerning",
+            WorkId::KernPairs => "BeKerning",
+            WorkId::KernSegment(..) => "BeKern",
+            WorkId::Kerns => "BeKernGather",
```

Comment:
the non-matching between WorkId variant name and identifiers makes me very confused